app/models/payment.rb

```diff
@@ -0,0 +1,41 @@
+class Payment < ApplicationRecord
+  # See: https://github.com/rails/rails/issues/31234#issuecomment-446629054
+  # Because of the open enum-state a payment needs to be able to be in (Mollie requirement),
+  # we're undefining #open before defining it with the enum.
+  class << self
+    undef_method :open
+  end
+  # See: https://docs.mollie.com/payments/status-changes
+  enum status: { open: 0, pending: 1, paid: 2, failed: 3, canceled: 4, expired: 5 }
+  COMPLETE_STATUSES = %w[paid failed canceled expired].freeze
+
+  belongs_to :user
+
+  validates :user, :status, presence: true
+  validates :amount, numericality: { greater_than: 20 }, presence: true
+
+
+  scope :uncompleted, (-> { where.not(status: COMPLETE_STATUSES) })
```

Comment:
```suggestion
  scope :not_completed, (-> { where.not(status: COMPLETE_STATUSES) })
```

Is more explicit. Otherwise `incomplete` is the correct English term :p